src/server/seeds/development/02_products.js

```diff
@@ -128,6 +129,7 @@ exports.seed = function (knex) {
           price: 120,
           color: 'Black',
           size: 'Large',
+          status: 'running_low',
```

Comment:
for id 11 , **status** should be default **out of stock** , according to issue #147

Reply:
okay i will update out_of_stock for 4 and 11.. and i will make stock to 0.